src/instance.ts

```diff
@@ -18,8 +18,9 @@
 
 const arrify = require('arrify');
 const common = require('@google-cloud/common-grpc');
-const {paginator} = require('@google-cloud/paginator');
-const {promisifyAll} = require('@google-cloud/promisify');
+import { ServiceObjectConfig } from '@google-cloud/common';
```

Comment:
nit for spaces

Reply:
Will fix when we turn the formatter back on :)